src/components/overlays/SavePrompt.tsx

```diff
@@ -0,0 +1,48 @@
+import classnames from 'classnames';
+import _ from 'lodash';
+import React from 'react';
+import { FiArrowRight } from 'react-icons/fi';
+import { getLabel } from '~/codeLists/labelCodes';
+import * as s from './savePrompt.scss'; // should use savePrompt.scss
+
+const SavePrompt = (props: any) => {
+    const changesContent = () => {
+        const current = _.cloneDeep(props.current);
+        const old = _.cloneDeep(props.old);
+        const divs = [];
+
+        for (const property in current) {
+            const linkAttribute = current[property];
+            const oldLinkAttribute = old[property];
+            if (_.isEqual(linkAttribute, oldLinkAttribute)) {
+                delete old[property];
+                delete old[property];
+            }
+        }
+
+        for (const property in old) {
```

Comment:
Do not use for loop and push to divs list, its ugly..

Instead use .map, like this:
```
return properties.map((property: string) => {
    return (
            <div className={classnames(s.formItem, s.savePromptRow)}>
             .....
            </div>
    )
}
```

Reply:
Also you can use a method? renderChangeRow?

```
return properties.map((property: string) => {
     const oldString = ...
     const newString = ...
     return this.renderChangeRow(oldString, newString);
}
```